core/settings.py

```diff
@@ -80,29 +56,21 @@
 
 WSGI_APPLICATION = 'core.wsgi.application'
 
-
-# Database
-# https://docs.djangoproject.com/en/5.1/ref/settings/#databases
-
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': os.environ.get("DB_NAME"),
-        'USER': os.environ.get("DB_USER"),
-        'HOST': os.environ.get("DB_HOST"),
-        "PASSWORD": os.environ.get("DB_PASSWORD"),
-        'PORT': os.environ.get("DB_PORT"),
+        'NAME': DBConfig.POSTGRES_NAME,
+        'USER': DBConfig.POSTGRES_USER,
+        'HOST': DBConfig.POSTGRES_HOST,
+        'PASSWORD': DBConfig.POSTGRES_PASSWORD,
+        'PORT': DBConfig.POSTGRES_PORT,
+        'POSTGRES_DB': DBConfig.POSTGRES_DB,
```

Comment:
```suggestion
        'DB': DBConfig.POSTGRES_DB,
```
```suggestion
        'DATABASE': DBConfig.POSTGRES_DB,
```
И опять же касательно конфига - я бы абстрагировался от postgres, мало ли когда-то мы захотим на другую бд перейти. тогда в нейминге каша будет. HOST, PORT и др. переменные в рамках DBConfig нам будут итак говорить, что мы хост, порт и др. переменные для бд тянем